apps/categories2/stylesheets/index.styl

```diff
@@ -1,5 +1,43 @@
-.categories-page
+@require '../../../components/stylus_lib'
+
+.l-categories-page
   padding 40px 0
 
-// To be added
-// .categories-page__header
+.l-categories-page__nav
+  display inline-block
+  width 25%
+  vertical-align top
+
+.l-categories-page__content
+  display inline-block
+  width 75%
+  vertical-align top
+
+.categories-page__header
+  garamond-size('xl-headline')
+
+.categories-page__blurb
+  garamond-size('s-headline')
+  margin 35px 0
+
+.categories-list
+  display block
+
+.categories-list__family
+  display block
+
+.categories-list__family__name
```

Comment:
Quickly to note here—it might never be a problem, but given this app is `categories2` you might want to consider namespacing by `categories2-list` just in case you accidentally naming things similar to the old app. The way we break up asset packages can tend to avoid this, but it's a good habbit to just always namespace by containing app or component folder name.